src/helpers/terminal_helpers.sh

```diff
@@ -214,3 +214,10 @@ function run_command {
   echo_header "$header"
   eval "$cmd" 2>&1
 }
+
+
+# Run a command without printing any output
+function run_command_silently {
+  local cmd="$1"
+  (run_command "$cmd") > /dev/null 2>&1
```

Comment:
I thought that's possible without a subshell. See https://github.com/Originate/git-town/blob/master/src/helpers/configuration_helpers.sh#L82